### ISSUE TITLE
Updated outdated content from bug bash in docs.microsoft.com for Service Bus

### ISSUE THREAD

**Issue by AlexGhiondea** [Azure/azure-sdk-for-java#17306]
- [Next steps under Queues Quick Starts](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-java-how-to-use-queues#next-steps)
   - The API reference link is wrong. Same in [Next steps under Topics Quick Starts](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-java-how-to-use-topics-subscriptions#next-steps)
   -  The "see more samples on Github" link is not needed as it is the same link as a few lines above in that section

**Comment by hemanttanwar** [comment]
PR : https://github.com/MicrosoftDocs/azure-docs/pull/67581/files

**Comment by ramya-rao-a** [comment]
@hemanttanwar Am thinking we update the "Next Steps" in the Java Quick Starts for both Queues and Topics/Subscriptions as below

- The samples should point to https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-java/servicebus-samples/ 
- The API reference docs should point to https://azuresdkdocs.blob.core.windows.net/$web/java/azure-messaging-servicebus/7.0.0/index.html

Thoughts?

**Comment by hemanttanwar** [comment]
I created PR for above change. once this PR is merged, we can close this issue.
https://github.com/MicrosoftDocs/azure-docs-pr/pull/141232

**Comment by ramya-rao-a** [comment]
@hemanttanwar https://github.com/MicrosoftDocs/azure-docs-pr/pull/141232 is for queues, are you planning another PR for topics?

**Comment by hemanttanwar** [comment]
Created new PR which include both queues and topics https://github.com/MicrosoftDocs/azure-docs-pr/pull/141241
